enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -19,20 +19,24 @@ Features currently supported:
 - embedding dashboards - static
 - embedding dashboards - w/drill-down
 - embedding the collection browser
+- ability for the user to modify existing questions
 - theming with CSS variables
-- plugins for custom actions
+- plugins for custom actions, overriding dashboard card menu items
+- subscribing to events
```

Comment:
What kind of "events" are we referring to here? If we're referring to dashboard loading events, then this is already supported in the latest release.

Reply:
Yes, that's why it's listed under features supported. Now?

Reply:
@albertoperdomo oh sorry, I thought this was in the "planned" section - the GitHub diffs are a bit misleading. all good then.